lib/Conversion/TorchToLinalg/Linear.cpp

```diff
@@ -549,32 +549,51 @@ class ConvertAtenConvolutionOp : public OpConversionPattern<AtenConvolutionOp> {
                                          "only support constant int dilations");
 
     Value N = getDimOp(rewriter, loc, input, 0);
+    Value IC = getDimOp(rewriter, loc, input, 1);
     SmallVector<Value> inDims;
     for (size_t i = 2; i < inRank; i++)
       inDims.push_back(getDimOp(rewriter, loc, input, i));
     Value F = getDimOp(rewriter, loc, weight, 0);
+    Value WC = getDimOp(rewriter, loc, weight, 1);
```

Comment:
F is the batch size for the weight, as per `linalg::Conv2DNchwFchw`. IC and WC are the same when groups are 1, but they're different otherwise. The constraint in that case is that they are both divisible by groups.